### PR TITLE
Changed URL to datacite supported EZ API

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by praneethpr** [comment]
Please use the test credentials for EZ Datacite API

**Comment by praneethpr** [comment]
Please use test credentials provided for Datacite EZ API

**Comment by praneethpr** [comment]
You can check the doi status on the web
https://doi.datacite.org/ (using the same credentials)

**Comment by praneethpr** [comment]
Add the url to your DOIutil.ini file
url = 'https://ez.datacite.org'

File: src/Pelagos/Util/DOIutil.php [hunk 1]
```diff
@@ -47,6 +54,7 @@ public function __construct()
         $this->doishoulder = $parameters['doi_api_shoulder'];
         $this->doiusername = $parameters['doi_api_user_name'];
         $this->doipassword = $parameters['doi_api_password'];
+        $this->url = $parameters['url'];
```
Comment (fightingtexasaggie) [inline src/Pelagos/Util/DOIutil.php]: perhaps something more descriptive, such as doi_api_url as our parameters files have a lot in them

**Comment by snguyen1** [comment]
You may want to add the URL parameter into the DOIutil.ini.example (for open-sourcing purposes)

**Pull request merged**